### PR TITLE
Add Coverage Not Decided to Coverage Status filter [STTNHUB-198] [STTNHUB-203]

### PR DESCRIPTION
**Pull request by devketanpro**
https://github.com/superdesk/superdesk-stt/pull/40
https://github.com/superdesk/superdesk-planning/pull/1844

### CODE REVIEW

File: newsroom/agenda/agenda.py [hunk 1]
```diff
@@ -547,11 +547,23 @@ def _filter_terms(filters, item_type):
...
+                must_term_filters.append(
+                    nested_query(
+                        path="coverages",
+                        query={
+                            "bool": {"filter": [{"terms": {"coverages.coverage_status": ["coverage not decided yet"]}}]}
+                        },
+                        name="coverage_status",
+                    )
+                )
             else:
```
Comment (petrjasek) [inline newsroom/agenda/agenda.py:560]: would be good to check the value here explicitly

File: assets/agenda/components/AgendaCoverageExistsFilter.tsx [hunk 2]
```diff
@@ -50,6 +53,12 @@ function AgendaCoverageExistsFilter ({toggleFilter, activeFilter}: any) {
                 onClick={() => toggleFilter(filter.field, FILTER_VALUES.NOT_PLANNED)}
             >{gettext('Coverage not planned')}
             </button>
+            <button
+                key='coverage-not-planned'
```
Comment (MarkLark86) [inline assets/agenda/components/AgendaCoverageExistsFilter.tsx:57]: This key should be unique

File: assets/agenda/components/AgendaCoverageExistsFilter.tsx [hunk 3]
```diff
@@ -25,6 +26,8 @@ function getActiveFilterLabel(filter: any, activeFilter: any) {
         return gettext('Planned');
     case FILTER_VALUES.NOT_PLANNED:
         return gettext('Not Planned');
+    case FILTER_VALUES.MAY_BE:
+        return gettext('May Be');
```
Comment (MarkLark86) [inline assets/agenda/components/AgendaCoverageExistsFilter.tsx:30]: This is the text that appears in the parent button of the dropdown, which has a different label to the option provided in the dropdown. 
I'm wondering if this should rather be `Not Decided` so the option in the dropdown list and the dropdown button text are relatable for end users

File: assets/agenda/components/AgendaCoverageExistsFilter.tsx (hunk 2 above)
Comment (devketanpro) [reply assets/agenda/components/AgendaCoverageExistsFilter.tsx:57]: my bad, I forgot this

File: assets/agenda/components/AgendaCoverageExistsFilter.tsx (hunk 3 above)
Comment (devketanpro) [reply assets/agenda/components/AgendaCoverageExistsFilter.tsx:30]: yep, sounds identical

**Review by MarkLark86 — APPROVED** [review]

**Pull request merged**